libs/ui/hover-card/brain/src/lib/brn-hover-card-content.service.ts

```diff
@@ -211,40 +229,45 @@ export class BrnHoverCardTriggerDirective implements OnInit, OnDestroy {
 		tap((visible) => visible && this._contentService.setState('open')),
 		switchMap((visible) => {
 			// we are delaying based on the configure-able input
-			return of(visible).pipe(delay(visible ? this.showDelay : this.hideDelay));
+			return of(visible).pipe(delay(visible ? this.showDelay() : this.hideDelay()));
 		}),
 		switchMap((visible) => {
 			// don't do anything when we are in the process of showing the content
 			if (visible) return of(visible);
 			// we set the state to closed here to trigger any animations for the element leaving
 			this._contentService.setState('closed');
 			// then delay to wait for the leaving animation to finish
-			return of(visible).pipe(delay(this.animationDelay));
+			return of(visible).pipe(delay(this.animationDelay()));
 		}),
 		distinctUntilChanged(),
 		share(),
 		takeUntil(this._destroy$),
 	);
 
-	@Input()
-	public showDelay = 300;
-	@Input()
-	public hideDelay = 500;
-	@Input()
-	public animationDelay = 100;
-	@Input()
-	public sideOffset = 5;
-
-	@Input()
-	public align: 'top' | 'bottom' = 'bottom';
+	public showDelay = input(300);
+	public hideDelay = input(500);
+	public animationDelay = input(100);
+	public sideOffset = input(5);
+	public align = input<'top' | 'bottom'>('bottom');
 
-	@Input()
-	public set brnHoverCardTriggerFor(value: TemplateRef<unknown> | BrnHoverCardContentDirective) {
-		this._contentService.setContent(value, this._vcr);
-	}
+	public readonly brnHoverCardTriggerForInput = input<TemplateRef<unknown> | BrnHoverCardContentDirective | undefined>(
+		undefined,
+		{ alias: 'brnHoverCardTriggerFor' },
```

Comment:
The new Styleguide for angular says that we should avoid alias. 
https://angular.dev/style-guide#avoid-aliasing-inputs-and-outputs
This was also added in the recommended eslint rules.

Reply:
okay, I will change that

Reply:
@goetzrobin Feedback on that would be nice as well :)

Reply:
That is a good point, but I think we just can use it without the Input suffix. 
My guess is that we just have to get used to this pattern, or we find other ways to deal with such issues.

Reply:
Yes I know, but then you have to think about it very carefully where you should use which variable, the linked signal is a good solution there we can do it in two steps and then we can call the first value state and us that because its writeable :)